isis/tests/FunctionalTestsCamdev.cpp

```diff
@@ -13,7 +13,7 @@ using namespace Isis;
 
 static QString APP_XML = FileName("$ISISROOT/bin/xml/camdev.xml").expanded();
 
-TEST_F(LineScannerCube, FunctionalTestsCamDevDefault) {
+TEST_F(LineScannerCube, FunctionalTestCamDevDefault) {
```

Comment:
If I'm following the normalization, shouldn't this be "FunctionalTestCamdevDefault" ?